packages/services/src/services/VideoPlayer/VideoPlayer.js

```diff
@@ -193,6 +193,17 @@ class VideoPlayerAPI {
     const fireEvent = this.fireEvent;
     return await this.checkScript().then(() => {
       const promiseKWidget = new Promise(resolve => {
+        let isCustomCreated;
+
+        if (!document.getElementById(targetId)) {
```

Comment:
@RaphaelAmadeu I'm wondering if we should check if `!document.getElementById(targetId)` as well as if `document.querySelector('dds-tabs-extended-media')` exists? That way in any weird cases where the target ID doesn't exist, it won't just append a video to the end of the page and not do anything else after that (hopefully that's an edge case but still...)

Reply:
yeah, I think it would be good! I'm adding this in my new commit

Reply:
Thanks @RaphaelAmadeu , looks great!